spec/controllers/sessions_controller_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe Users::SessionsController, type: :controller do
+  login_user
+  let(:user) { subject.current_user }
+
+  controller do
```

Comment:
could it be moved to some support class? doesn't look like something we need in this file